ex8/phonebook.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="style/style.css">
+  <script data-main="js/config" src="js/lib/require/require.js"></script>
+  <script type="application/javascript">
+    window.onload = function() {
+      require(['config'], function() {
+        require(['ajax','update'], function() {
+          ajax_get('json/data.json', function(data) {
```

Comment:
Style error: insertJson (camelCase) vs. ajax_get (all lower case  with underscore). Please use camelCase (standard for javascript).